File: tracker_benchmark/trackers/CXT/run_CXT.m
function results=run_CXT(seq, res_path, bSaveImage)

close all;

x=seq.init_rect(1)-1;%matlab to c
y=seq.init_rect(2)-1;
w=seq.init_rect(3);
h=seq.init_rect(4);

path = './results/';

if ~exist(path,'dir')
    mkdir(path);
end

tic
command = ['CXT.exe 1 0 0 1 ' seq.name ' ' seq.path ' ' path ' '   num2str(seq.startFrame) ' ' num2str(seq.endFrame) ' '  num2str(seq.nz) ' ' seq.ext ' ' num2str(x) ' ' num2str(y) ' ' num2str(w) ' ' num2str(h)];

dos(command);
duration=toc;

results.res = dlmread([path seq.name '_CXT.txt']);   
results.res(:,1:2) =results.res(:,1:2) + 1;%c to matlab
length(results.res)

results.type='rect';
results.fps=seq.len/duration;

results.fps = dlmread([path seq.name '_CXT_FPS.txt']);

File: tracker_benchmark/trackers/CXT/run_CXT.py
import subprocess
import sys

def run_CXT(tracker, exe, args):
    path = os.path.abspath(TRACKER_SRC + tracker)
    path = os.path.join(path, exe)
    command = [path] + map(str, args)
    subprocess.call(command)

if __name__ == "__main__":
    # tracker = 'CXT'
    # exe = 'CXT.exe'
    # seqs = butil.load_all_seq_configs()
    seqs = [butil.load_seq_config('Coke'), butil.load_seq_config('Bolt')]
    # resultPath = './results/';
    # for seq in seqs:
    #     x = seq.gtRect[0][0] - 1
    #     y = seq.gtRect[0][1] - 1
    #     w = seq.gtRect[0][2]
    #     h = seq.gtRect[0][3]
    #     args = ['1', '0', '0', '1', seq.name, seq.path, resultPath,
    #         seq.startFrame, seq.endFrame, seq.nz, seq.ext, x, y, w, h]
    #     run_tracker_win(tracker, exe, args)
    #     